src/components/AboutSection/style.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.section`
+    padding: 0 3rem;
+    background:  linear-gradient(180deg, #4979B7 0%, #3C457B 65.85%, #393A6E 100%), linear-gradient(180deg, #4979B7 0%, #393A6E 100%);
+    border-radius: 25px;
+    color: #FFF;
+ 
+    & > p {
+        font-size: 1.5rem;
```

Comment:
acho que ficou grande, podia ser:
```suggestion
        font-size: 1.25rem;
```